tests/spring/src/test/java/org/jboss/fuse/wsdl2rest/test/calculator/CalculatorDocLitTest.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * #%L
+ * Wildfly Camel :: Testsuite
```

Comment:
this license seems strange. Why wildfly Camel?

Reply:
Just went with the same header as was included on the MenardsDocLitTest to be honest.

Reply:
I think it would be better to use the classic Apache license header:
/**
 * Licensed to the Apache Software Foundation (ASF) under one or more
 * contributor license agreements.  See the NOTICE file distributed with
 * this work for additional information regarding copyright ownership.
 * The ASF licenses this file to You under the Apache License, Version 2.0
 * (the "License"); you may not use this file except in compliance with
 * the License.  You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

Reply:
We would need to change that throughout the rest of the code I suspect?

Reply:
Not necessarily, if the code was copy-pasted from elsewhere with this license header, I think it needs to be kept.
For new files, the Apache license seems more suitable.